docs/content/documentation/systems/AuxKernels/framework/ConstantAux.md

```diff
@@ -1,6 +1,7 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # ConstantAux
+
+The Constant AuxKernel creates a constant field over the domain. You should consider using a GeneralPostprocessor for truly constant fields, however the ConstantAux value can be controlled by the Control system.
+
```

Comment:
I don't like the second sentence. A GeneralPostprocessor if not a field variable. Perhaps you can state:

```
Truly constant field variables can likely be replaced with a [Postprocessor](/Postprocessors/index.md).
```

Also, what about mentioning the constant material property stuff.